pkg/test/dependency/dependencies.go

```diff
@@ -18,8 +18,11 @@ const (
 	// Apps is a dependency on fake networked apps. This can be used to mimic traffic thru a mesh.
 	Apps = Instance("apps")
 
+	// Cluster is required for running the test.
+	Cluster = Instance("cluster")
```

Comment:
Currently `cluster` implies `kubernetes cluster`, correct? If so, maybe we should be more specific with our terminology?

Reply:
Kube?